xcell/mappers/mapper_base.py

```diff
@@ -24,6 +27,10 @@ def _get_defaults(self, config):
         self.mask_power = config.get('mask_power', 1)
         self.coords = config['coords']
         self.mask = None
+        self.signal_map = None
+        # Defined here for the signal_map file name
+        self.zbin = None
+        self.dndz = None
```

Comment:
Humm, why do you need `zbin` and `dndz`? These seem too specific to be generally useful.

Reply:
To be able to add them to the file names in the previous format (after 'signal_map')

Reply:
I didn't see `dndz` being used

Reply:
probably it was another equivalent name to `zbin` that I forgot to remove

Reply:
you have to declare `dndz` because we have the `_get_shifted_nz` method which uses it.